CppSamples/Maps/ShowDeviceLocationUsingIndoorPositioning/IndoorsLocationDataSourceCreator.cpp

```diff
@@ -117,10 +117,5 @@ void IndoorsLocationDataSourceCreator::findPathwaysTable()
 
 void IndoorsLocationDataSourceCreator::returnIndoorsLocationDataSource()
 {
-  // The IndoorsLocationDataSource constructor takes an optional GlobalId to identify which row of the positioning table to use.
-  // If not specified, the constructor will use the row from the most recent survey
-  if (m_globalId.isNull())
-    emit createIndoorsLocationDataSourceCompleted(new IndoorsLocationDataSource(m_positioningTable, m_pathwaysTable, this));
-  else
-    emit createIndoorsLocationDataSourceCompleted(new IndoorsLocationDataSource(m_positioningTable, m_pathwaysTable, m_globalId, this));
+    emit createIndoorsLocationDataSourceCompleted(new IndoorsLocationDataSource(m_positioningTable, m_globalId, this));
```

Comment:
The code is written in a way to protect against an empty globalId. I assume that can still happen, but I am not sure. Seems like this will probably do something wrong if that does happen.

No action required here as there's unlikely any other reasonable change we can make here without a bigger refactor.